.github/pull_request_template.md

```diff
@@ -37,6 +39,15 @@ In each section, please describe design decisions made, including:
 
 <hr>
 
+##### Key changed/added classes in this PR
+ * `MyFoo`
+ * `OurBar`
+ * `TheirBaz`
+
+<hr>
+
+<!-- Check the items by putting "x" in the brackets for the done things. Not all of these items apply to every PR. Remove the items which are not done or not relevant to the PR. None of the items from the checklist above are strictly necessary, but it would be very helpful if you at least self-review the PR. -->
```

Comment:
nit: comment references "checklist above", but with this change the checklist is now below

Reply:
Thanks, fixed it.